README.md

```diff
@@ -0,0 +1,128 @@
+# GKE Service Accounts Initializer
+
+> :warning: **This is pre-release software. Do not use it for production
+> purposes.**
+
+This add-on inject [Google Cloud Service Accounts][sa] credentials, which are
+previously imported to the cluster as Secrets, to [Pods] running on Container
+Engine.
+
+[sa]: https://cloud.google.com/compute/docs/access/service-accounts
+[Pods]: https://kubernetes.io/docs/concepts/workloads/pods/pod/
+
+Install this initializer to your cluster and add the following annotation to
+metadata.annotations field of your Pod template or specification:
+
+```yaml
+annotations:
+  iam.cloud.google.com/service-account: "[SECRET-NAME]"
+```
+
+## Quickstart
+
+Create an alpha cluster on GKE (Initializers feature is not beta until v1.9):
+
+    gcloud container clusters create test-cluster \
+        --enable-kubernetes-alpha \
+        --cluster-version 1.7.5
```

Comment:
1.7.5 doesn't seem to be a valid version, check out this https://screenshot.googleplex.com/s3tMJxsohgP.

Reply:
redacted internal link as this page is public. thanks for caching.